### ISSUE TITLE
Breaks admin site for Django 2.1

### ISSUE THREAD

**Issue by andreyshipilov** [edoburu/django-any-imagefield#7]
Hi everyone. Basically I get this error in admin:
```
TypeError at /admin/work/poster/add/
render() got an unexpected keyword argument 'renderer'
```

However, if I comment field in the model, everything works fine. The image field is not shown of course.

Any ideas?

**Comment by vdboor** [comment]
yes, the `render()` method signature changed in Django 2.1.

You may want to see if you can fix that: https://github.com/edoburu/django-any-imagefield/blob/master/any_imagefield/forms/widgets.py#L12 and see whether that needs some extra care in Django 2.1. It might take some time before I get to it.

**Comment by johncpang** [comment]
Just change two lines here, add `renderer` after `attrs`:

```python
Line 12: def render(self, name, value, attrs=None, renderer=None):
Line 23: input_field = super(ImagePreviewWidget, self).render(name, value, attrs, renderer)
```

**Comment by vdboor** [comment]
Thanks, I've fixed this and released v1.0
